shell/src/main/java/alluxio/cli/fsadmin/command/ReportCommand.java

```diff
@@ -172,17 +186,27 @@ public int run(CommandLine cl) throws IOException {
               mBlockMasterClient, mPrintStream);
           capacityCommand.run(cl);
           break;
+        case CONFIGURATION:
+          ConfigurationCommand configurationCommand = new ConfigurationCommand(
+              mMetaMasterClient, mPrintStream);
+          configurationCommand.run();
+          break;
         case SUMMARY:
-          SummaryCommand summaryCommand = new SummaryCommand(mMetaMasterClient,
-              mBlockMasterClient, mPrintStream);
+          SummaryCommand summaryCommand = new SummaryCommand(
+              mMetaMasterClient, mBlockMasterClient, mPrintStream);
           summaryCommand.run();
           break;
+        case UFS:
+          UfsCommand ufsCommand = new UfsCommand(mFileSystemMasterClient);
+          ufsCommand.run();
+          break;
         default:
           break;
       }
     } finally {
-      mMetaMasterClient.close();
       mBlockMasterClient.close();
+      mFileSystemMasterClient.close();
```

Comment:
consider using Guava closer to close multiple resources.
The reason is the current code may fail to close the second object if an exception is thrown when closing the 1st. So Guava closer will always enforce all resources are closed

Reply:
@apc999 Would it be strange to initialize closer in the constructor? Thanks for introducing the Guava closer to me!

```java
private final Closer mCloser;

public ReportCommand() {
    mCloser = Closer.create();
    MasterClientConfig config = MasterClientConfig.defaults();
    mBlockMasterClient = mCloser.register(new RetryHandlingBlockMasterClient(config));
    mFileSystemMasterClient = mCloser.register(new RetryHandlingFileSystemMasterClient(config));
    mMetaMasterClient = mCloser.register(new RetryHandlingMetaMasterClient(config));
    mPrintStream = System.out;
}

public int run(CommandLine cl) throws IOException {
    try {
    } finally {
      mCloser.close();
    }
}
```